Moto/E5/res/values/config.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<bool name="config_enableAutoPowerModes">true</bool>
+	<integer name="config_autoPowerModeAnyMotionSensor">30</integer>
+	<bool name="config_wifi_dual_band_support">false</bool>
+	<bool name="config_wifi_fast_bss_transition_enabled">true</bool>
+	<bool name="config_wifi_background_scan_support">true</bool>
+	<bool name="config_wifi_enable_disconnection_debounce">true</bool>
+	<bool name="config_automatic_brightness_available">true</bool>
+	<integer name="config_brightness_ramp_rate_fast">200</integer>
+	<integer name="config_brightness_ramp_rate_slow">40</integer>
+	<bool name="config_setColorTransformAccelerated">true</bool>
+	<bool name="config_suspendWhenScreenOffDueToProximity">true</bool>
+
+	<integer-array name="config_longPressVibePattern">
+		<item>0</item>
+		<item>25</item>
+	</integer-array>
+	<integer-array name="config_virtualKeyVibePattern">
+		<item>0</item>
+		<item>34</item>
+	</integer-array>
+	<integer-array name="config_keyboardTapVibePattern">
+		<item>0</item>
+		<item>25</item>
+	</integer-array>
+	<bool name="config_hotswapCapable">true</bool>
+	<integer name="config_screenBrightnessSettingMinimum">2</integer>
+	<integer name="config_screenBrightnessSettingDefault">90</integer>
+	<integer name="config_screenBrightnessDoze">17</integer>
+	<integer name="config_autoBrightnessBrighteningLightDebounce">2000</integer>
+	<integer name="config_screenBrightnessDark">2</integer>
+	<integer-array name="config_autoBrightnessLcdBacklightValues">
+		<item>2</item>
+		<item>10</item>
+		<item>35</item>
+		<item>55</item>
+		<item>65</item>
+		<item>74</item>
+		<item>90</item>
+		<item>90</item>
+		<item>90</item>
+		<item>90</item>
+		<item>112</item>
+		<item>134</item>
+		<item>150</item>
+		<item>158</item>
+		<item>213</item>
+		<item>255</item>
+		<item>255</item>
+		<item>255</item>
+		<item>256</item>
```

Comment:
That sounds wrong, but if you tell me it works...

Reply:
i thought the same, but works